flathunt.py

```diff
@@ -31,7 +31,7 @@ def launch_flat_hunt(config, heartbeat: Heartbeat):
 
     wait_during_period(time_from, time_till)
 
-    hunter = Hunter(config, id_watch)
+    hunter = OpenAIHunter(config, id_watch)
```

Comment:
Why do you want to create an `OpenAIHunter` here, instead of just adding `generate_text` to the filters for the default `Hunter`?

Reply:
As another idea: I could introduce some additional logic in `generate_text()` for this. I was thinking something like
```python
    def generate_text(self):
        """Add processor to generate text, if enabled"""
        if self.config.openai_enabled():
            if not any([isinstance(processor, CrawlExposeDetails) for processor in self.processors]):
                self.crawl_expose_details()
            self.processors.append(OpenAIProcessor(self.config))
        return self
```

How about this to prevent the need for a new hunter?